pyproject.toml

```diff
@@ -49,7 +49,10 @@ repository = "https://github.com/spacetelescope/roman_datamodels"
 roman_datamodels = "roman_datamodels.stnode._integration:get_extensions"
 
 [build-system]
-requires = ["setuptools >=61", "setuptools_scm[toml] >=3.4", "wheel"]
+requires = [
+    "setuptools >=61",
+    "setuptools_scm[toml] >=3.4",
+]
```

Comment:
Why is `wheel` dropped?

Reply:
Wheel is unnecessary now because setuptools now always builds a wheel. It's been like that for 5+ years now apparently.